src/ruby/yast/wfm.rb

```diff
@@ -213,6 +213,61 @@ def self.call_builtin_wrapper(*args)
             "Caller:  #{e.backtrace.first}"
     end
 
+    private_class_method def self.handle_signal_exception(e)
+      signame = Signal.signame(e.signo)
+      msg = "YaST receive signal %s and will exit.\n If killing of YaST is " \
+        "not requested then please report a bug report with logs obtained from " \
+        "save_y2logs script." % signame
```

Comment:
- How does this message look in a 80x25 terminal?
- "YaST receive**d** **a** %s signal and will exit"
- What do you mean "if killing of YaST is not requested"? I think it depends on the signal, and this is the part that could annoy users the most. We have to get this detail right.